insert_data_into_mysql.py

```diff
@@ -0,0 +1,46 @@
+import textstat
+import text_analysis as ta
+import mysql.connector as mysql
+
+my_db = mysql.connect (
+    user = "leakraus",
+    password = "LeaK",
+    database = "gunning_fog_index"
+)
+my_cursor = my_db.cursor() 
+
+def insert_data_in_mysql(text:list):
+    for i in range(len(text)):
+        if key_already_exists_in_database(text[i]['key']) == False:
+            key = text[i]["key"]
+            title = text[i]["title"]
+            description_index = textstat.gunning_fog(text[i]["description"])
+            if text[i]["response"] == None:
+                response_index = 0
```

Comment:
Ich würde hier None nehmen, dann wird einfach kein Wert in der DB gespeichert.
Eventuell muss man in der Datenbank dann noch Null-Values erlauben.
```suggestion
                response_index = None
```